spec/rubocop/cop/style/conditional_assignment_assign_to_condition_spec.rb

```diff
@@ -1080,26 +1082,32 @@
                                 '      end'].join("\n"))
     end
 
-    it 'corrects assignment in if elsif else' do
-      source = ['if foo',
-                '  bar = 1',
-                'elsif baz',
-                '  bar = 2',
-                'else',
-                '  bar = 3',
-                'end']
+    shared_examples '1 character assignment types' do |asgn|
```

Comment:
Yeah, I really don't know what to call this cop anymore. It already supports `=~`, `!~`, and `<=>` which aren't assignments. `+` and `-` make sense in the spirit of what the cop is doing, but not its name. 

Given what it is currently doing, it could also make sense to add support for `==`, `>`, `<`, `>=`, and `<=`. At that point, I think the cop would be handling more comparisons than assignments.